.github/workflows/ci.yml

```
@@ -44,66 +44,12 @@ jobs:
         id: npm-ci-test
         run: npm run ci-test
 
-  test-action-linux:
-    name: GitHub Actions Test Linux
-    runs-on: ubuntu-latest
-
-    steps:
-      - name: Checkout
-        id: checkout
-        uses: actions/checkout@v4
-
-      - name: Test Local Action
-        id: test-action
-        uses: ./
-        with:
-          pkl-version: 0.26.0
-          github-token: ${{ secrets.GITHUB_TOKEN }}
-
-      - name: Confirm download
-        run: pkl --version
-
-  test-action-macos-amd64:
-    name: GitHub Actions Test MacOS amd64
-    runs-on: macos-13
-
-    steps:
-      - name: Checkout
-        id: checkout
-        uses: actions/checkout@v4
-
-      - name: Test Local Action
-        id: test-action
-        uses: ./
-        with:
-          pkl-version: 0.26.0
-          github-token: ${{ secrets.GITHUB_TOKEN }}
-
-      - name: Confirm download
-        run: pkl --version
-
-  test-action-macos-aarch64:
-    name: GitHub Actions Test MacOS aarch64
-    runs-on: macos-14
-
-    steps:
-      - name: Checkout
-        id: checkout
-        uses: actions/checkout@v4
-
-      - name: Test Local Action
-        id: test-action
-        uses: ./
-        with:
-          pkl-version: 0.26.0
-          github-token: ${{ secrets.GITHUB_TOKEN }}
-
-      - name: Confirm download
-        run: pkl --version
-
-  test-action-windows:
-    name: GitHub Actions Test Windows
-    runs-on: windows-latest
+  test-action-different-os:
```

```suggestion
  test-action:
```

Github already makes it pretty easy to see what the matrix value was, this seems less verbose.

---

Done [here](https://github.com/pkl-community/setup-pkl/pull/39/commits/50a4d9157e1a5576bb59d97320231e22b81f518a).
Thanks for the suggestion!